src/mixins/legal-api-mixin.ts

```diff
@@ -61,7 +61,7 @@ export default class LegalApiMixin extends Vue {
    * @param isDraft Boolean indicating whether to complete filing.
    */
   private createFiling (data: object, isDraft: boolean): Promise<any> {
-    let url = ''
+    let url = 'https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/businesses'
```

Comment:
This needs to be a property from configuration.json (locally) or config map (deployed). Same as in Business Filings.

Reply:
I did this so i could post to businesses as the URL thats set in the config map has the `/` on the end which is invalid to a request made directly to businesses.

Reply:
My idea was to change the urls in the config map, but for congruency they were appended with the /. I'm open to any suggestions tho!

Reply:
Well, we could remove "businesses" from the config key and add it every time we use it, either with no slash or with a slash and other params.
It would make the LEGAL_API consistent with the other URL definitions.
Eg,
```
"LEGAL_API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/"
```

Reply:
If you go this route, add a comment to the release ticket to update the config map in the different Openshift environments.

Reply:
Ok sounds great! Will update this.

Reply:
This has been updated. Worth another look tho! If your ok with this, this PR can be merged 👍